java/cli/build.gradle

```diff
@@ -16,11 +16,11 @@ repositories {
     jcenter()
     //mavenLocal()
     mavenCentral()
-    maven {
-        url 'http://artifacts.openmicroscopy.org/artifactory/ome.releases'
+    omeMaven {
```

Comment:
You're absolutely correct - https://docs.gradle.org/current/userguide/artifact_dependencies_tutorial.html#defineRemoteMavenRepo. Will push imminently